code/game/objects/items/storage/boxes.dm

```diff
@@ -34,6 +34,23 @@
 	w_class = SIZE_LARGE //Changed becuase of in-game abuse
 	storage_flags = STORAGE_FLAGS_BOX
 
+/obj/item/storage/box/pride
+	name = "box of prideful crayons"
+	desc = "A box of every flavor of pride."
+	storage_slots = 8
+	w_class = SIZE_SMALL
+	can_hold = list(/obj/item/toy/crayon/pride)
```

Comment:
When inherited, this doesn't need to change. In other words, any crayon should be able to be held in the box.